Cargo.toml

```diff
@@ -26,19 +26,18 @@ cpal-asio = ["cpal/asio"]
 dropseed-core = { path = "./core" }
 meadowlark-core-types = "0.3"
 audio-graph = { git = "https://github.com/MeadowlarkDAW/audio-graph", branch = "main" }
-clack-host = { git = "https://github.com/prokopyl/clack", rev = "d2c0f2e43b56e7037cb96729cebfb40fbc868c51" }
-clack-extensions = { git = "https://github.com/prokopyl/clack", features = ["clack-host", "audio-ports", "log", "note-ports", "params", "state", "thread-check"], rev = "d2c0f2e43b56e7037cb96729cebfb40fbc868c51" }
+clack-host = { git = "https://github.com/prokopyl/clack", rev = "782bbfce72f6b3505dc70807bbebadcb3dc5ac1b" }
+clack-extensions = { git = "https://github.com/prokopyl/clack", features = ["clack-host", "audio-ports", "log", "note-ports", "params", "state", "thread-check"], rev = "782bbfce72f6b3505dc70807bbebadcb3dc5ac1b" }
 dirs = { version = "4.0", optional = true }
 basedrop = "0.1"
-smallvec = "1.7"
+smallvec = { version = "1.9.0", features = ["const_generics", "union"] }
```

Comment:
What's the advantage of using const_generics and unions here?

Reply:
[`const_generics`](https://docs.rs/smallvec/1.9.0/smallvec/#const_generics) allows to use buffers of any size, and probably helps a little bit with compile times since it's a single impl vs. a large amount of macro-generated impls for each array size (to a point). 

[`union`](https://docs.rs/smallvec/1.9.0/smallvec/#union) is the most useful of the two since it makes `SmallVec`s smaller by 1 word (8 bytes on 64bit) by removing the internal enum tag, which I found stacks up quite fast since we're using `SmallVec`s of `SmallVec`s to store buffer pointers.